templates/layout/region--header.html.twig

```diff
@@ -13,7 +13,22 @@
         {{ elements.utility_navigation }}
       {% endblock %}
       {% block site_header__search %}
-        {{ drupal_block('views_exposed_filter_block:search-page_1', {label_display: false}, false) }}
+        <form action="/search" method="get" id="header-search-form" accept-charset="UTF-8">
+          <div class="form-item form-item-keywords">
+            <label for="edit-keywords--header-search-form">
+              {% include "@atoms/images/icons/_yds-icon.twig" with {
+                icon__name: 'magnifying-glass',
+                icon__blockname: utility_nav__base_class,
+                icon__modifiers: ['search'],
+                icon__decorative: true,
+              } %}
+            </label>
+            <input placeholder="Search this site" type="text" id="edit-keywords--header-search-form" name="keywords" value="" size="30" maxlength="128" class="form-text">
+          </div>
+          <div class="form-actions form-wrapper" id="edit-actions--header-search-form">
+            <input class="visually-hidden button form-submit" type="submit" id="edit-submit-search--header-search-form" value="Search">
+          </div>
+        </form>
```

Comment:
This should be captured (and probably simplified) in the component library. As a rule, we don't write markup (HTML) in our Drupal templates. They should just be referencing markup files in our component library.

This also doesn't currently work on mobile, so something needs to change. And if we're going to write custom markup anyway, there's already an example form in Storybook that does have the correct styles applied (it was intended to be a placeholder, but if we're going that route, it probably just needs a few tweaks.)

Reply:
This work has been completed in [PR-68](https://github.com/yalesites-org/atomic/pull/68)